packages/create-cloudflare/templates/hello-world-durable-object/js/src/index.js

```diff
@@ -16,25 +18,27 @@
  */
 
 /** A Durable Object's behavior is defined in an exported Javascript class */
-export class MyDurableObject {
+export class MyDurableObjec extends DurableObject {
 	/**
 	 * The constructor is invoked once upon creation of the Durable Object, i.e. the first call to
-	 * 	`DurableObjectStub::get` for a given identifier
+	 * 	`DurableObjectStub::get` for a given identifier (no-op constructors can be omitted)
```

Comment:
what do you mean that they can be omitted? can they? 😕

Reply:
The constructor is there so that people will know where to start, but see the [counter](https://developers.cloudflare.com/durable-objects/examples/build-a-counter/) example (no pun intended on counter lol).

Reply:
ah wow, ok that's my bad! I was under the impression that you could not omit the constructor! 🙂👍